corehq/apps/README.rst

```diff
@@ -0,0 +1,191 @@
+data_dictionary
+   The data dictionary documents a project's data model, specifically, its case types and case properties.
+   This makes it possible to reference those definitions throughout the app-building process.
+   The data dictionary is used partially for project documentation but is also referenced in a few other
+   parts of HQ: for example, when configuring case properties to be updated in a form, app manager will
+   pull the properties' descriptions from the data dictionary.
+data_interfaces
+   Functionality for taking a few specific actions on project data, such as reassigning cases in bulk.
+   Most of this app relies heavily on standard reporting functionality.
+fixtures
+   "Fixtures" is used generally in the codebase to refer to datasets sent to the phone, which includes the
+   casedb, mobile report data, etc., but the fixtures app primarily deals with one specific type of fixture,
+   which is lookup tables.
+groups
+   Users can be assigned to groups for the purposes of sharing cases within a group and for reporting purposes.
+hqadmin
+   Internal admin functionality used by the development, support, QA, and product teams.
+linked_domain
+   Functionality to share certain configuration data between domains: apps, lookup tables, report definitions, etc.
+   Work is done in a primary "upstream" domain, and then that domain's data models are copied to one or more
+   "downstream" domains. This is most often used to set up a development => production workflow, where changes are made
+   in the development domain and then pushed to the production domain, where real project data is entered.
+   Linked domains are also used by certain enterprise-type projects that manage one program across multiple regions
+   and use a separate downstream domain for each region.
+registration
+   Workflows for creating new accounts on HQ.
+reminders
+   A subset of SMS/messaging, including functionality around incoming SMS keywords. "Reminders" is a leftover term from a previous iteration of the messaging framework.
+saved_reports
+   Functionality to let users save a specific set of report filters and optionally run reports with those filters on a scheduled basis.
+toggle_ui
+   Framework for feature flags, which are used to limit internal feature to specific domains and/or users.
+translations
+   Functionality for managing application translations, including integration with Transifex, which is used by a small number of projects.
+user_importer
+   Bulk importing of users.
+
+Tertiary Apps
+^^^^^^^^^^^^^
+These apps may be useful parts of the system but don't have as much active development as the groups above.
+
+aggregate_ucrs
+   TODO
+analytics
+   Integrations with third-party analytics tools such as Google Analytics and Kissmetrics.
+   Also contains internal product-focused tools such as AB testing functionality.
+builds
+   Models relating to CommCare Mobile builds, so that app builders can control which mobile version their apps use.
```

Comment:
Also somewhat legacy since a large part of the functionality was related to J2ME builds right?

Reply:
some of it, yes, I'll add a note

Reply:
b641ad69c2a93a5815630acea1f124b681d395cc